app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -134,9 +131,11 @@ export class ContractsController {
 
     for(let i = 0; i < contractsData.length; i++){
       const contract = contractsData[i],
-            final_amount = (contract.final_amount === '' || contract.final_amount === undefined) ? 0.0 : parseFloat(contract.final_amount);
+            final_amount = (contract.final_amount === '' || contract.final_amount === undefined) ? 0.0 : parseFloat(contract.final_amount),
```

Comment:
contract could be `null` or `undefined`? You may initialize previous line as `const contract = contractsData[i] || {}` or use destructuring.

Wait, comma-separated declarations? No way, repeat `const` keyword for each declaration. Easier readibility.

Reply:
`contract` shouldn't be `null` or `undefined`